gi-gtk-hs/src/Data/GI/Gtk.hs

```diff
@@ -1,20 +1,22 @@
 module Data.GI.Gtk
     ( module GI.Gtk
-    , module Data.GI.Gtk.ModelView.Types
+    , module Data.GI.Gtk.ComboBox
     , module Data.GI.Gtk.ModelView.CellLayout
     , module Data.GI.Gtk.ModelView.CustomStore
-    , module Data.GI.Gtk.ModelView.SeqStore
     , module Data.GI.Gtk.ModelView.ForestStore
+    , module Data.GI.Gtk.ModelView.SeqStore
     , module Data.GI.Gtk.ModelView.TreeModel
-    , module Data.GI.Gtk.ComboBox
+    , module Data.GI.Gtk.ModelView.Types
+    , module Data.GI.Gtk.Widget
     ) where
 
 import GI.Gtk hiding (treeModelGetValue, treeModelGetIter)
 import Data.GI.Gtk.BuildFn
-import Data.GI.Gtk.ModelView.Types
+import Data.GI.Gtk.ComboBox
 import Data.GI.Gtk.ModelView.CellLayout
 import Data.GI.Gtk.ModelView.CustomStore
-import Data.GI.Gtk.ModelView.SeqStore
 import Data.GI.Gtk.ModelView.ForestStore
+import Data.GI.Gtk.ModelView.SeqStore
 import Data.GI.Gtk.ModelView.TreeModel
-import Data.GI.Gtk.ComboBox
+import Data.GI.Gtk.ModelView.Types
+import Data.GI.Gtk.Widget
```

Comment:
These lists have both been sorted alphabetically, and `Data.GI.Gtk.Widget` has been added.